src/style.css

```diff
@@ -5,21 +5,30 @@
 
 body {
   font-family: Arial, Helvetica, sans-serif;
+  height: 90vh;
```

Comment:
[OPTIONAL] I would suggest you add functionality to display `scores` when we open the webpage as well. It would be a great feature if you add it. You can use this method [`Windows onload MDN`](https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event).